spec/controllers/course/controller_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe Course::Controller, type: :controller do
+  describe '#sidebar' do
+    it 'returns an empty array when no modules included' do
+      expect(Course::Controller.new.sidebar).to eq([])
```

Comment:
Actually my specs here is a bit nasty.. any other way to get the instance of  controller ?